src/js/index.js

```diff
@@ -0,0 +1,104 @@
+import {
+  duplicationOperator,
+  maxLength,
+  plus,
+  minus,
+  division,
+  multiplication,
+  equal,
+  zero,
+} from "./utils/define.js";
+
+const numberBtn = document.getElementById("number");
+const totalCount = document.getElementById("total");
+const clearBtn = document.getElementById("clear");
+const operatorBtn = document.getElementById("operator");
+
+const isNaNCustom = (value) => Number.isNaN(Number(value));
+const totalCountToArr = () => totalCount.textContent.split("");
+const valueList = (operator) => totalCount.textContent.split(operator);
+const isOperator = (value) => isNaNCustom(value);
+const curOperator = () => totalCountToArr().find(isOperator);
+
+const isFirstNodeIsZero = () => {
+  const [firstNode] = totalCountToArr();
+  return firstNode === zero;
+};
+
+const isLessThanMaxValue = () => {
+  const MAX_VALUE = 1000;
+  const curCount = totalCount.textContent;
+  if (curOperator()) {
+    const [_, second] = valueList(curOperator());
+    return Number(second) < MAX_VALUE;
+  }
+  return Number(curCount) < MAX_VALUE;
+};
+
+const addNumber = (cur) => {
+  if (isFirstNodeIsZero()) {
+    totalCount.textContent = "";
+  }
+
+  totalCount.appendChild(document.createTextNode(cur));
+
+  const isErr = !isLessThanMaxValue() && !isOperator(cur);
+  if (isErr) {
+    onErrHandler(maxLength);
+    totalCount.removeChild(totalCount.lastChild);
+  }
+};
+
+const onErrHandler = (type) => {
+  switch (type) {
+    case duplicationOperator:
+      alert("숫자를 먼저 입력한 후 연산자를 입력해주세요!");
+      break;
+    case maxLength:
+      alert("숫자는 세 자리까지만 입력 가능합니다!");
+      break;
+    default:
+      return;
+  }
+};
+
+const roundDown = (value) =>
+  value >= 0 ? Math.floor(value) : Math.ceil(value);
```

Comment:
약간의 편법(?)으로 다음과 같이 표현할 수 있답니다.
```suggestion
const roundDown = (value) => ~~value;
```

`~` 연산은 bit 연산인데, 1의 보수 연산을 수행해줍니다.
이 과정에서 모든 소수점을 버리게 됩니다.
`~~`처럼 표현할 경우 1의 보수 연산을 2번 수행합니다. 즉, 소수점만 버리고 원래대로 되돌려놓는거죠 ㅎㅎ

그런데 숫자가 매우 커질 경우에는 제대로 작동하지 않을 때가 있습니다.